artifacts/deploy/karmada-metrics-adapter.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: karmada-metrics-adapter
+  namespace: karmada-system
+  labels:
+    app: karmada-metrics-adapter
+    apiserver: "true"
+spec:
+  selector:
+    matchLabels:
+      app: karmada-metrics-adapter
+      apiserver: "true"
+  replicas: 1
```

Comment:
I think yes, but it could be multiple replicas

Reply:
Please set it to at least two replicas. The reason is that we have a lesson learned before, that `we guess it could work with multiple replicas, but it doesn't`.